configure

```diff
@@ -7994,4 +7996,3 @@ fi
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: Good, the configure finished." >&5
 printf "%s\n" "$as_me: Good, the configure finished." >&6;}
 printf "%s\n"
-
```

Comment:
restore this line then everything is good.

Reply:
>restore this line

I'm sorry, I don't understand?!

Reply:
Re-add an empty line after line 7998.

Reply:
The empty line is there but it seems to get removed on push or something?

![configure_empty_line](https://user-images.githubusercontent.com/42386382/181502504-333085ec-2e70-469e-926f-e727fe39d75d.png)

Reply:
Or notepad++ just _shows_ that there's an empty line in the end but there isn't really.

Reply:
normal notepad shows it too?? as in cursor is on next line underneath `printf "%s\n"`

Reply:
I don't know why this file has 2 newlines in the end. 1 is standard for POSIX AFAIK. The point is that this file is generated by a script and even if one newline is removed now, it will be re-added the next time this file is updated. Currently (before this PR) i see 2 newlines in the end in kate and qt creator.